### DISCUSSION TITLE
Can the Ulanzi Pixel Clock be flashed?

### DISCUSSION THREAD

**Discussion by aptonline** [pixelit-project/PixelIt#258] [Q&A] [answered]
Is it possible to flash the Ulanzi pixel clock with PixelIt firmware?

**Comment by aptonline** [comment, marked as answer]
Found this in the discord which seems to suggest it can be it at lest being tested: https://discord.com/channels/558849582377861122/572925721836519436/1068835701677891584

Happy to do any testing (as long as doesn’t brick anything 🤣)

**Comment by matze20211** [comment]
I try to flash  firmware_v2.2.0_wemos_d1_mini32.bin  webui is shown, no safe config, a sound always to hear. is there no PixelIT bin for Ulanzi available?

**Comment by psycor** [comment]
Searching for files to flash the ulanzi clock.
Discord seems to be unavailabel.
All i try was a fault.
Some solution ?

**Comment by matze20211** [comment]
Hy, look at Discord [https://discord.com/channels/558849582377861122/572925721836519436/1076594760711671838](https://discord.com/channels/558849582377861122/572925721836519436/1076594760711671838):
apt: As there is no binary you’ll need to use Visual Studio Code to flash. Make sure you also install the platformio extension as well and also make sure you flash the file system too so you can save config changes made via the webUI.

**Comment by foorschtbar** [comment]
Ulanzi is supported with Release [2.4.3](https://github.com/pixelit-project/PixelIt/releases/tag/2.4.3)
